### PR TITLE
chore(build): new ami based on amazon linux 2023 💥 - breaking change

### PR DESCRIPTION
**Pull request by mariusgheorghies**
💥 breaking change: the new AMI will have TCP ILP disabled by default and have no tools pre-installed to generate auth key

- new ami based on amazon linux 2023
- disable UDP ILP by default
- disable TCP ILP by default
- data volume default to type gp3

### CODE REVIEW

**Comment by questdb-butler** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please update the PR title to match this format:
`type(subType): description`

Where `type` is one of:
`feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, `test`, `ci`, `revert`

And: `subType` is one of:
`build`, `sql`, `log`, `mig`, `core`, `ilp`, `pgwire`, `http`, `conf`, `ui`, `wal`

For Example:

```
perf(sql): improve pattern matching performance for SELECT sub-queries
```
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against dbb9bf830e4e181c87f3c2562ba4ef5a97aa2cbf
</p>

**Review by bluestreak01 — DISMISSED** [review]

File: pkg/ami/marketplace/assets/server.conf [hunk 1]
```diff
@@ -516,8 +516,8 @@ line.udp.enabled=true
 
 ######################### LINE TCP settings ###############################
 
-line.tcp.enabled=true
-line.tcp.auth.db.path=conf/auth.txt
+line.tcp.enabled=false
```
[the remark this answers is not in the document]
Comment (mariusgheorghies) [reply pkg/ami/marketplace/assets/server.conf]: enabled back here: [375e55a](https://github.com/questdb/questdb/pull/4519/commits/375e55a7a3665e9be53887ab68a4408c81ca73b7)

[the remark this answers is not in the document]
Comment (bluestreak01) [reply pkg/ami/marketplace/assets/server.conf]: lets disable again

[the remark this answers is not in the document]
Comment (mariusgheorghies) [reply pkg/ami/marketplace/assets/server.conf:519]: done in: [2f7594d](https://github.com/questdb/questdb/pull/4519/commits/2f7594d7b7124e0f1e79159a97108be76053ae49)

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**